atomic_reactor/plugins/exit_store_metadata.py

```diff
@@ -61,46 +61,40 @@ def get_digests(self):
         """
 
         digests = {}  # repository -> digest
-        for registry in self.workflow.push_conf.docker_registries:
+        if self.workflow.conf.registry:
```

Comment:
Do we need these types of conditions?

Reply:
not really, it was only there for keeping a few edge cases working where registry wasn't defined but now with 'minItems' set we don't need it. removed